### DISCUSSION TITLE
creating new UNO solution in Rider fails

### DISCUSSION THREAD

**Discussion by drei97** [unoplatform/uno#17701] [Q&A] [answered]
Hi,

trying to create UNO project/solution in Rider (using the new plugin) leads to the following error

![image](https://github.com/user-attachments/assets/41003330-db9b-4cb6-b836-18409c2c4693)

This is on Linux!

I am not sure if the error is related to the UNO plugin or to Rider itself.

Joachim

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This is expected as it is a bug in Rider. You can follow Rider issues here: https://github.com/unoplatform/uno/issues/15226.

To create projects, you can follow our [getting started guide](https://platform.uno/docs/articles/get-started-rider.html) which uses the [Live Wizard](https://new.platform.uno/).
